app/src/main/java/com/github/scribeWizTeam/scribewiz/util/Export.kt

```diff
@@ -0,0 +1,66 @@
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+import androidx.core.content.FileProvider
+import java.io.File
+
+class Export {
+    companion object {
```

Comment:
Should replace hardcoded strings with constants and reference them

Reply:
Thank you I will do a sweep of this in a later branch and make sure there are no hard coded strings anywhere.